src/ocean/net/ssl/openssl/OpenSsl.d

```diff
@@ -0,0 +1,833 @@
+/******************************************************************************
+
+    C binding to OpenSSL
+
+    copyright: Copyright (c) 2018 dunnhumby Germany GmbH. All rights reserved
+
+
+    The OpenSSL library is very large. This binding includes only a tiny
+    fraction of the available functions.
+
+*******************************************************************************/
+
+module ocean.net.ssl.openssl.OpenSsl;
+
+
+import core.stdc.config : c_ulong, c_long;
+import ocean.transition;
+
+
+extern (C)
```

Comment:
`extern(C):` to remove an useless level of indentation ?

Reply:
Well, there are things at the end of the file which are NOT `extern(C)`.

Reply:
You can explicitly mark them as `extern(D)` to override default